README.rst

```diff
@@ -118,11 +118,12 @@ You can also request help by submitting a question on StackOverflow.
 Building
 ========
 
-You can install this package in a virtual environment like so::
+You can install this package -- after cloning -- in a virtual environment like so::
 
     $ python3 -m venv /path/to/venv
     $ source /path/to/venv/bin/activate
-    $ python setup.py install
+    $ python -m pip install -U pip setuptools
+    $ python -m pip install .
```

Comment:
I think here we should definitely recommend editable mode, if we're suggesting folks clone flr dev. Otherwise they can just install from pypi

Reply:
Same as before. Let me do some study / experiments.

Reply:
@pepoluan feel free to ask me if you have questions about Python packaging. I maintain PyPUG.